src/base/model.js

```diff
@@ -125,6 +125,9 @@ ModelBase.prototype.idAttribute = 'id';
  * to {@link Model#hasTimestamps hasTimestamps}.  The first element will
  * be the created column name and the second will be the updated
  * column name.
+ * You can pass values for the timestamps columns as parameter in the
+ * {@link Model#save save} method. This will prevent the automatic
+ * updated of the timestamps columns values.
```

Comment:
Replace `updated` with `update`.

I think you should further clarify that passing in custom values for timestamps will use those values instead. Something like:

> ... of the timestamps columns values, and use the values you specify instead.

It's better to be too obvious than not obvious enough.